configuration-example.yaml

```diff
@@ -47,43 +48,10 @@ locations:
   esUrl: http://elastic-search-url.example.com
   esIndex: elastic-search-index
   estype: elastic-search-type
+  esIndexService: elastic-search-services-index
+  estypeService: elastic-search-services-type
+  gatewayUrl: https://api.example.com/v1/
```

Comment:
I left a comment in the configs repo about this. I was wondering why you're adding back gatewayUrl since we already have endpointUri. I think we changed the code a few months ago to remove gatewayUrl dependencies.